capstone-frontend/src/components/ListPage.jsx

```diff
@@ -115,14 +122,23 @@ class ListPage extends Component {
               {this.state.bookList.userID === window.localStorage.getItem('userID') ?
                 <Col className='margin-auto p-0 mr-3'>
                   <div id='modal-buttons' className='mx-2'>
+                    <UpdateList
+                      id='button-list-add'
+                      btnStyle='btn btn-primary h-100'
+                      bookListId={this.props.id}
+                      updateListPage={this.updateListPage}
```

Comment:
Just wondering if you needed `updateListPage` inside `UpdateList` (couldn't find where it was used)?

Reply:
This was the way I originally refreshed the entire page on name change, but I realized it was less expensive to just have a setter for the title that I passed instead of refetching the entire patch. Thanks for reminding me!